Feature/Main HTML Markup

miraclmoore opened the pull request:
I have started on 2 of the 3 sections of the Main HTML Markup for the site. 

I have separated the site into 3 Sections to make it easier for myself:
Section 1: Blog or Blog Content
Section 2: What We Do or Services
Section 3: Testimonials

There are still some things I need to address:
1. Are we using the blog text, links, pics, etc...from the site? 
(Currently Section 1 just has placeholders, blank info, and Lorem Ipsum Text)
2. I just coded the body without integrating what Willa and Camper have coded. Is that not the process?
3. I'm not so good with figuring out the columns and bootstrap so that still needs to be fixed for Section 1
4. I need to figure out how to create the widget at the end of Section 1 as well.

willatauro committed:
Create Dummy1

willatauro committed:
Merge pull request #28 from chingu-voyage3/willastaging

Create Dummy1

miraclmoore committed:
feat(main): started on Main HTML Markup for the site

miraclmoore committed:
Merge pull request #1 from chingu-voyage3/staging

Pulling index.html

miraclmoore committed:
Update to Main HTML Markup

I have removed some of the unclosed tags, and am working through trying to integrate Flexbox to create the 2 colums for the blogposts. I also added in the pics for the blogposts

camper-fcc commented on index.html:
```diff
@@ -32,6 +29,139 @@ <h1 class="company-name">Eleva<span>tion</span></h1>
 
+
+<!--BODY, BUILT BY CMOORE-->
+<body>
```
The opening `<body>` tag needs to be below the closing `</head>` tag and above the `<!--HEADER, BUILT BY CAMPER-->` comment on line 14. If we merge this code the way it is now, with the opening body tag where it is, it'll overwrite the correct code currently in the staging branch, essentially creating a bug.

camper-fcc commented on index.html:
```diff
@@ -32,6 +29,139 @@ <h1 class="company-name">Eleva<span>tion</span></h1>
+<body>
+
+<!--SECTION ONE BLOG CONTENT--> 
+<section class="a" id="blog">
```
What does the `a` class mean? Can you make this more descriptive so we know what it's for? If it's a placeholder, it's better to remove the `class="a"` code entirely.

miraclmoore commented:
Hey @camper-fcc! I'll make these changes and submit a new PR. I couldn't figure out how to delete the "Dummy1" file though.

miraclmoore committed:
Made changes to index.html 

Updated portions of the index.html per the review from fcc--camper.

miraclmoore committed:
Updates to Index.html

Removed lines 172-180, positions the bottom BODY tag under the FOOTER tag, Also added in the MAIN tag and nested the SECTIONS inside of them.

miraclmoore committed:
Removed More Buggy Code From Index.html

Removed 4 more bugs from the index.html file. We are now bug free people!

miraclmoore committed:
Removed Dummy File and Resized Headings

Removed the Dummy1 file that was added mistakenly, as well as resized the headers to make the text easier to navigate.

shalinidc reviewed (approved):
Looks good to me 👍🏻

miraclmoore committed:
Seriously Deleting the Dummy1 File
